gcp-resources/README.md

```diff
@@ -14,41 +14,37 @@ The following OpenTelemetry semantic conventions will be detected:
 | host.id                 | auto                                         | auto                                                   |                                          |                                                |                                                |
 | host.name               | auto                                         | auto                                                   |                                          |                                                |                                                |
 | host.type               | auto                                         | auto                                                   |                                          |                                                |                                                |
-| k8s.pod.name            |                                              | downward API or auto                                   |                                          |                                                |                                                |
-| k8s.namespace.name      |                                              | downward API                                           |                                          |                                                |                                                |
-| k8s.container.name      |                                              | hardcoded (manual)                                     |                                          |                                                |                                                |
+| k8s.pod.name            |                                              | environment variable                                   |                                          |                                                |                                                |
+| k8s.namespace.name      |                                              | environment variable                                   |                                          |                                                |                                                |
```

Comment:
Aren't these still using the downward API ?

Reply:
@dashpole could you shed some light here ? 

Going through the [downward API documentation](https://kubernetes.io/docs/concepts/workloads/pods/downward-api/) did not clear up what would be apt to put in the table. 

I think downward API explicitly means [downward API volume](https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/).

Reply:
These should use the downward API to set an environment variable (e.g. following https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/). Then the user needs to set OTEL_RESOURCE_ATTRIBUTES to use the environment variables produced by the downward API.

Reply:
(k8s.container.name is still a hardcoded environment variable), rather than one set by the downward API.

Reply:
Thanks for the explanation @dashpole. It makes sense.

Regarding the k8s.container.name, IIUC, the user would still have to pass the value of the environment variable manually to the OTEL_RESOURCE_ATTRIBUTES for proper resource detection, correct ?

Just setting the an environment variable named "CONTAINER_NAME" would not be enough.

Reply:
yes, that is correct